### PR TITLE
Continue report screen

### PR DESCRIPTION
**Pull request by adnanjt**

### CODE REVIEW

**Comment by adnanjt** [comment]
Current status
image

File: app/components/DR/CalendarButton/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          dateInput: {
+            borderWidth: 0,
+            alignItems: 'center'
+          },
+          dateText: { color: '#6B6B6B', fontSize: 16 }
+        }}
+      />
+    </View>
+  );
+}
```
Comment (JoseFcoEstrella) [inline app/components/DR/CalendarButton/index.js]: Without line break

File: app/components/DR/Input/index.js [hunk 2]
```diff
@@ -30,4 +30,4 @@ export default function Input({
       />
     </View>
   );
-}
+}
```
Comment (JoseFcoEstrella) [inline app/components/DR/Input/index.js]: WLB

File: app/components/DR/PhoneInput/index.js [hunk 3]
```diff
@@ -42,4 +42,4 @@ export default function PhoneInput({
       keyboardType="number-pad"
     />
   );
-}
+}
```
Comment (JoseFcoEstrella) [inline app/components/DR/PhoneInput/index.js]: WLB

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM. Just need fix some line break

**Review by RutsSantos — CHANGES_REQUESTED** [review]
LGTM. **But, solve conflicts.**
Also, i see React-native version is 0.62.2 when the MIT project is built in RN 0.61.5, wouldn't that make any trouble?

**Comment by adnanjt** [comment]
> LGTM. **But, solve conflicts.**
> Also, i see React-native version is 0.62.2 when the MIT project is built in RN 0.61.5, wouldn't that make any trouble?

I can try fixing this today if I have time We should use the same one but not sure why updated 

GOING TO FIX THIS ON NEXT PR PLS APPROVE

**Review by inteljudajocu — CHANGES_REQUESTED** [review]
images asset missing, the same asset as the other PR

**Comment by adnanjt** [comment]
> images asset missing, the same asset as the other PR

This is gonna b fixed on the other one

**Review by inteljudajocu — APPROVED** [review]

**Pull request merged**